CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to `otp_builds`
+
+1. [License](#license)
+1. [Reporting a bug](#reporting-a-bug)
+1. [Requesting or implementing a feature](#requesting-or-implementing-a-feature)
+1. [Submitting your changes](#submitting-your-changes)
+   1. [Code Style](#code-style)
+   1. [Committing your changes](#committing-your-changes)
+   1. [Pull requests and branching](#pull-requests-and-branching)
+
+## License
+
+`otp_builds` is licensed under [Apache License Version 2.0](LICENSE.md) License, for all code.
+
+## Reporting a bug
+
+`otp_builds` is not perfect software and will be buggy.
+
+Bugs can be reported via
+[GitHub issues: bug report](https://github.com/erlef/otp_builds/issues/new?template=bug_report.md).
```

Comment:
We don't have a bug report template. I don't think we need it either, let's see what kind of issues we will be getting and adjust then.

Reply:
Not that it matters much, but it's "just naming". The link still works and would adapt to a potential future template.